src/output.cpp

```diff
@@ -1146,6 +1146,23 @@ void popup_nowait( const char *mes, ... )
     popup( text, PF_NO_WAIT );
 }
 
+void popup_status( const char *title, const char *msg, ... )
```

Comment:
> Why does the function require separate text and title strings?

So that the output to `stderr` is:
```
Loading core data
Loading mods (foo)
Loading mods (bar)
```

and not:
```
Please wait while the world data loads...
Loading core data
Please wait while the world data loads...
Loading mods (foo)
Please wait while the world data loads...
Loading mods (bar)
```

> why isn't the title expected to be valid?

Checking `title` for `nullptr` allows the caller to omit a title if they don't need one.

> It currently writes to std::cerr twice: here and in the main popup function

Only if `test_mode` is set, I've pushed a fix for that

> Finally: popup( text += fmt, - why +=, why not just +?

Fixed